README.rst

```diff
@@ -90,9 +90,19 @@ The built docs will be in ``docs/_build/html/``, to view them go to ``file:///pa
     
 
 Release Protocol
-^^^^^^^^^^^^^^^^
+================
 
-Follow the `Astropy template release instructions <https://docs.astropy.org/en/latest/development/astropy-package-template.html>`_. 
+GitHub Action Releases
+^^^^^^^^^^^^^^^^^^
+
+The `pypi-package.yml` GitHub workflow is built to make a package release off the latest commit hash in `main`. The job in this workflow is triggered when a
+release tag is generated and pushed to `main`, and uses `OpenAstronomy`'s `GitHub action workflow <https://github.com/OpenAstronomy/github-actions-workflows>`_
+for publishing pure Python packages (`see here <https://github-actions-workflows.openastronomy.org/en/stable/publish_pure_python.html>`_ for documentation).
```

Comment:
```suggestion
The `pypi-package.yml <.github/workflows/pypi-package.yml>`_ GitHub workflow creates a PyPI release. The job in this workflow is triggered when a tag is pushed or a GH release (+tag) is created, and uses `OpenAstronomy`'s `GitHub action workflow <https://github.com/OpenAstronomy/github-actions-workflows>`_
for publishing pure Python packages (`see here <https://github-actions-workflows.openastronomy.org/en/stable/publish_pure_python.html>`_ for documentation).
```

Reply:
trying to write rst links by hand so i might have gotten the formatting wrong there ^^